server/client_test.go

```diff
@@ -2061,32 +2061,6 @@ func TestClientNoSlowConsumerIfConnectExpected(t *testing.T) {
 	}
 }
 
-func TestClientStalledDuration(t *testing.T) {
```

Comment:
Is it worthwhile adding a new test by mocking up some `*client`s with custom `tst`, `pb` and `mp` values and then calling `stalledWait`?

Reply:
For what purpose?

Reply:
Mostly thinking that we don't have any unit tests that prove that `stalledWait` actually blocks for as long as we expect or when we'd expect, or that it unblocks when the `stc` is closed.